### ISSUE TITLE
tests routinely hang 

### ISSUE THREAD

**Issue by SethTisue** [typelevel/squants#356]
seen in the Scala community build ([example run](https://scala-ci.typesafe.com/view/scala-2.11.x/job/scala-2.11.x-integrate-community-build/1327/console))

also reproducible locally with `++2.11.12` followed by `test`:

```
  | => squants.space.SpaceChecks 286s
  | => squants.space.SpaceChecks 287s
  | => squants.space.SpaceChecks 287s
  | => squants.space.SpaceChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.market.MarketChecks 287s
  | => squants.experimental.json.QuantitySerializerSpec 288s
```

**Comment by SethTisue** [comment]
I've seen this on Scala 2.12, too

**Comment by cquiroz** [comment]
This should be fixed on #357 when making the tests not to run in parallel

**Comment by SethTisue** [comment]
let's try it: https://github.com/scala/community-build/pull/990
